lib/src/mediadevices_impl.dart

```diff
@@ -126,4 +127,65 @@ class MediaDevicesWeb extends MediaDevices {
         width: _mapConstraints['width'],
         zoom: _mapConstraints['zoom']);
   }
+
+  @override
+  Future<MediaDeviceInfo> selectAudioOutput(
```

Comment:
@davidliu It seems that all browsers have not implemented this method
https://developer.mozilla.org/en-US/docs/Web/API/MediaDevices/selectAudioOutput#browser_compatibility

<img width="747" alt="image" src="https://user-images.githubusercontent.com/17733746/182764711-fc44e06a-7f55-49a6-94ea-0f384d886f3d.png">

Reply:
Maybe we should throw an exception and prompt to use `MediaElement.setSinkId` instead.